lib/editor/components/HourMinuteInput.js

```diff
@@ -67,7 +68,7 @@ export default class HourMinuteInput extends Component<Props, State> {
   }
 
   render () {
-    const {seconds: initialSeconds, style, ...otherProps} = this.props
+    const {seconds: initialSeconds, standaloneInput, style, ...otherProps} = this.props
     const {hours, minutes} = this.state
 
     const defaultValue = !minutes && !hours ? '' : '00'
```

Comment:
Not sure if this is something we want to solve in this PR, but you can't enter `0` into an empty field.

Reply:
This is a good point since midnight should be a valid input. I have updated this in [0d258e6](https://github.com/ibi-group/datatools-ui/pull/892/commits/0d258e683ef66e3c08bc2cb25f47eafa001a3489) which changes the behaviour in the offset tool as well, but not significantly since it just allows offsetting by 0.